dev/com.ibm.ws.microprofile.health.2.0/bnd.bnd

```diff
@@ -38,11 +38,12 @@ Import-Package: \
   *
 
 Export-Package: \
-	com.ibm.ws.microprofile.health.resources,\
+	!com.ibm.ws.microprofile.health.resources,\
```

Comment:
By marking it private, it won't export.  So this line isn't needed.  Same comment for all of the files.

Reply:
Updated.